lib/Backend/amd64/LowererMDArch.cpp

```diff
@@ -1643,6 +1653,12 @@ LowererMDArch::GeneratePrologueStackProbe(IR::Instr *entryInstr, IntConstType fr
     // $done:
     //
 
+    if (this->m_func->IsTrueLeaf() &&
```

Comment:
@LouisLaf : What if the calling JS code is also small and doesn't probe ?

---
In reply to: [69667255](https://github.com/Microsoft/ChakraCore/pull/1239#discussion_r69667255) [](ancestors = 69667255)